setup.py

```diff
@@ -25,5 +25,8 @@
     download_url = "https://github.com/kszlim/osu-replay-parser/tarball/v" + __version__,
     license = "MIT",
     test_suite="tests",
+    install_requires=[
+        "osucore>=0.1.11",
+    ],
```

Comment:
this is pulling from pypi right now, right? Having a separate package for the rust parser seems like bad style. Is this the intended path forward for the actual release? How would releasing a new osrparse version go? Would we have to build and release a new osucore version first, then bump the osucore requirement in osrparse and build and release a new osrparse version?

Ideally this would all be self contained in one pypi package and require only one build/release to release a new version.